### PR TITLE
:wrench: Added CODEOWNERS file

### PR DESCRIPTION
**Pull request by devtobi**
**Description**
- Added CODEOWNERS file
- Added admin and maintainer group as CODEOWNERS for all files
- Added admin group as CODEOWNERS for CODEOWNERS file itself

**Reference**
Issues #97

### CODE REVIEW

**Review by simonhir — APPROVED** [review]
LGTM but has no effect as long as no branch rules are defined in my opinion

**Comment by devtobi** [comment]
> LGTM but has no effect as long as no branch rules are defined in my opinion

Thats correct, I will activate this after merging. ;)

File: .github/CODEOWNERS
Comment (simonhir) [inline .github/CODEOWNERS:2]: @devtobi is there a reason for mentioning admins and maintainers here?

René mentioned he is getting many notifications without really needing his attention, so only maintainers would be better fitting in my opinion.

Comment (devtobi) [reply .github/CODEOWNERS:2]: We can definitely change that. I wanted to admins be able to approve as well. Problem is, when we only set maintainers here, admins (even they have more rights than maintainers) will not be able to approve an MR in terms of code ownership.
If admins won't actively participate in development, we can of course remove those from the CODEOWNER file. However the CODE_OWNERS file itself should only be approvable by admins.

Comment (simonhir) [reply .github/CODEOWNERS:2]: As far as i know admins are mostly also added to the maintainer group.
I create a MR for that

**Pull request merged**
